### PR TITLE
Fts support search contact

### PR DESCRIPTION
**Pull request by Tougee**

### CODE REVIEW

File: app/src/main/java/one/mixin/android/ui/common/messenger/Messenger.kt [hunk 1]
```diff
@@ -208,8 +208,8 @@ class Messenger @Inject internal constructor(private val jobManager: MixinJobMan
         val transferContactData = ContactMessagePayload(shareUserId)
         val encoded = GsonHelper.customGson.toJson(transferContactData).base64Encode()
         val message = createContactMessage(
-            UUID.randomUUID().toString(), conversationId, sender.userId,
-            category, encoded, shareUserId, MessageStatus.SENDING.name, nowInUtc(), replyMessage?.messageId, replyMessage?.toQuoteMessageItem()
+            UUID.randomUUID().toString(), conversationId, sender.userId, category, encoded, shareUserId,
+                MessageStatus.SENDING.name, nowInUtc(), sender.fullName, replyMessage?.messageId, replyMessage?.toQuoteMessageItem()
```
Comment (SeniorZhai) [inline app/src/main/java/one/mixin/android/ui/common/messenger/Messenger.kt]: Here you should insert contact data,  not sender.

**Review by SeniorZhai — CHANGES_REQUESTED** [review]

File: app/src/main/java/one/mixin/android/ui/common/messenger/Messenger.kt (hunk 1 above)
Comment (Tougee) [reply app/src/main/java/one/mixin/android/ui/common/messenger/Messenger.kt]: fixed.

File: app/src/main/java/one/mixin/android/ui/search/SearchMessageAdapter.kt [hunk 2]
```diff
@@ -51,6 +51,9 @@ class SearchMessageHolder(itemView: View) : RecyclerView.ViewHolder(itemView) {
         if (message.type == MessageCategory.SIGNAL_DATA.name || message.type == MessageCategory.PLAIN_DATA.name) {
             TextViewCompat.setCompoundDrawablesRelative(itemView.search_msg_tv, icon, null, null, null)
             itemView.search_msg_tv.text = message.mediaName
+        } else if (message.type == MessageCategory.SIGNAL_CONTACT.name || message.type == MessageCategory.PLAIN_CONTACT.name) {
+            TextViewCompat.setCompoundDrawablesRelative(itemView.search_msg_tv, null, null, null, null)
+            itemView.search_msg_tv.text = message.mediaName
```
Comment (SeniorZhai) [inline app/src/main/java/one/mixin/android/ui/search/SearchMessageAdapter.kt:56]: Change a suitable name?

**Review by SeniorZhai — APPROVED** [review]

File: app/src/main/java/one/mixin/android/ui/search/SearchMessageAdapter.kt (hunk 2 above)
Comment (Tougee) [reply app/src/main/java/one/mixin/android/ui/search/SearchMessageAdapter.kt:56]: This field is compatible with multiple contents and needs to be considered later.

**Pull request merged**